crane-cloud/frontend
# Creating project tags on the front-end

# Description

This is to illustrate how the project tags will be created on the front-end, someone is able to create a tag and also remove it.

## Type of change


## Trello Ticket ID

https://trello.com/c/FKx1RvFI

## How Can This Been Tested?

Fetch the remote branch "ft/project-tags" and visit the project settings and move on to the update modal, there you see an input section for project tags. Add a specific tag to your project, make sure it is not blank. 

## Checklist:


# Screenshots

![image](https://github.com/crane-cloud/frontend/assets/110970478/70896bb7-e843-402e-b1d0-5079a31f80e1)
